corehq/apps/app_manager/models.py

```diff
@@ -2346,6 +2354,13 @@ def is_auto_select(self):
     def max_select_value(self):
         return self.case_details.short.max_select_value
 
+    def has_grouped_tiles(self):
+        return (
+            hasattr(self, 'case_details')
+            and self.case_details.short.case_tile_template
+            and self.case_details.short.case_tile_group.index_identifier
```

Comment:
Can't remember how couch works: is this going to fail if `case_tile_group` is None?

Reply:
No, because we wrap it with JsonObject will implicitly creates all these objects, even if they are not in the underlying data.